page199_question4.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+#step1: state number variable
+secret_number = 11
+
+#step2: get user input to guess the number
+user_guess = int(input("Guess my secret number: "))
+
+#step3: write one conditional statement re: if guess is too low
+if user_guess < 11:
```

Comment:
These conditionals are all good! You got the three possible conditions (too high; too low; and equals). 💯